imageio/meta.yaml

```diff
@@ -1,26 +1,27 @@
 package:
   name: imageio
-  version: 1.2
+  version: 1.3
 
 source:
-  fn: imageio-1.2-linux32.zip  [linux32]
-  url: https://pypi.python.org/packages/source/i/imageio/imageio-1.2-linux32.zip  [linux32]
+  fn: imageio-1.3-linux32.zip  [linux32]
+  url: https://pypi.python.org/packages/source/i/imageio/imageio-1.3-linux32.zip  [linux32]
```

Comment:
Yep, none of the above if Anaconda comes with freeimage as an independent library :-)

Well, maybe we can leave these source files for Windows, but definitely not for Mac and Linux.